src/main/scala/org/scalajs/tools/tsimporter/Main.scala

```diff
@@ -17,26 +17,19 @@ import parser.TSDefParser
 /** Entry point for the TypeScript importer of Scala.js */
 object Main {
   def main(args: Array[String]) {
-    if (args.length < 2) {
-      Console.err.println("""
-        |Usage: scalajs-ts-importer <input.d.ts> <output.scala> [<package>]
-        |  <input.d.ts>     TypeScript type definition file to read
-        |  <output.scala>   Output Scala.js file
-        |  <package>        Package name for the output (defaults to "importedjs")
-      """.stripMargin.trim)
-      System.exit(1)
-    }
+    Config.parser.parse(args, Config()) match {
+      case None => // do nothing. Scopt shows nice error message.
```

Comment:
Consider using
```scala
for (config <- Config.parser.parse(args, Config())) {
  ...
}
```
instead, since you're doing nothing in the `case None`.